x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/stats/RestEqlUsageTestCase.java

```diff
@@ -178,7 +178,7 @@ public void testEqlRestUsage() throws IOException {
                 "  [process where serial_event_id < 4] by exit_code" + 
                 "  [process where opcode == 1] by user" + 
                 "  [process where opcode == 2] by user" + 
-                "  [file where parent_process_name == 'file_delete_event'] by exit_code" +
+                "  [file where parent_process_name == \\\"file_delete_event\\\"] by exit_code" +
```

Comment:
Why `\\\"` and not just `\"`?

Reply:
It's because of the `runEql()` which constructs a json body, so with `\"` it's escaped in the java string but then it's interpreted as a closing `"` in json.